src/main/java/com/stripe/param/EphemeralKeyDeleteParams.java

```diff
@@ -0,0 +1,98 @@
+// File generated from our OpenAPI spec
```

Comment:
Why did this appear?

Reply:
Because it's now true and it wasn't before? There was a PR to remove `associatedObjects`.